README.md

```diff
@@ -62,10 +62,10 @@ The local chain is started at `PORT`, which defaults to 8545.
 
 The publish step of the app works exactly as `aragon apm publish`, described below.
 
-#### `aragon apm publish` or `apm publish`
+#### `aragon apm publish`
```

Comment:
Here's something from left field... what do you think about `aragon package` for the command?

Reply:
Sounds good, adding an alias for `aragon package` and if catches on we can maybe change or docs to recommend using it.